open-recipes/src/types/OR_Item.ts

```diff
@@ -0,0 +1,17 @@
+export interface OR_Item {
```

Comment:
```suggestion
export interface OpenRecipeServiceItem {
```

Would this be better for naming conventions?  OR i feel isn't that descriptive but lmk if I'm being too much lol